### DISCUSSION TITLE
why the character after the search string not highlight as red color?

### DISCUSSION THREAD

**Discussion by basheewang** [folke/flash.nvim#349] [General]
![6-17-2024 12-15-20 PM](https://github.com/folke/flash.nvim/assets/5795648/98bf95c0-4aa2-4c84-9f79-d840603cbe43)
I am only  using the default setup, and I am using NvChad 2.5 config.Please help.

**Comment by folke** [comment]
Because that's what your colorscheme shows. They should properly implement flash highlight groups

**Reply by basheewang** [reply]
What kind of colorscheme should I use to see the red highlight character?

**Reply by basheewang** [reply]
I am using tokyonight now and it shows correctly. :)
